WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -101,45 +102,67 @@
-                    is the recommended symbol icon font.<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
-            <TextBlock
-                Margin="0,8,0,0"
-                Style="{ThemeResource BodyStrongTextBlockStyle}"
-                Text="How do I get the font?" />
-            <RichTextBlock>
-                <Paragraph>
-                    On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
-                    On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>
-                    .<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
+        <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons Font is the recommended symbol icon font."/>
+        <labs:SettingsExpander Grid.Row="1"
+                  IsExpanded="False"
+                  Header="Instructions on how to use Segoe Fluent Icons">
+            <labs:SettingsExpander.Items>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <RichTextBlock TextWrapping="Wrap">
+                        <Paragraph FontWeight="SemiBold">
+                            How to get the font
+                        </Paragraph>
+                        <Paragraph>
+                            On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
+                            On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it
+                            <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>.
+                        </Paragraph>
+                    </RichTextBlock>
+                </labs:SettingsCard>
+                <labs:SettingsCard ContentAlignment="Left" Padding="16,8">
+                    <StackPanel Orientation="Vertical" Spacing="8">
+                        <RichTextBlock TextWrapping="Wrap">
+                            <Paragraph FontWeight="SemiBold">
+                                How to use the font
+                            </Paragraph>
+                            <Paragraph>
+                                If you don't specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run>,
+                                or you specify a <Run FontFamily="Consolas, monospace" FontWeight="Bold">FontFamily</Run> that is not available on the system at runtime,
+                                the <Hyperlink NavigateUri="https://learn.microsoft.com/uwp/api/windows.ui.xaml.controls.fonticon">FontIcon</Hyperlink> falls back to the default font family defined by the
+                                <Run FontFamily="Consolas, monospace" FontWeight="Bold">SymbolThemeFontFamily</Run> theme resource.
```

Comment:
Nit: according to the docs, it's used as a {StaticResource SymbolThemeFontFamily}
```suggestion
                                <Run FontFamily="Consolas, monospace" FontWeight="Bold">SymbolThemeFontFamily</Run> resource.
```

Reply:
I copied it from the MS website. So I don't know?

Reply:
Should be StaticResource, see SymbolThemeFontFamily section: https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font#using-the-icons